library/ssl_tls.c

```diff
@@ -1908,7 +1908,7 @@ int mbedtls_ssl_set_hs_ecjpake_password( mbedtls_ssl_context *ssl,
         return( MBEDTLS_ERR_SSL_HW_ACCEL_FAILED );
```

Comment:
Github doesn't let me comment on line 1899, so I'm commenting on the closest line I can: following the discussion about how to interpret "errors after the call to `psa_pake_setup()`" in the spec, I think we need to call abort() there too.